packages/core/useFullscreen/index.ts

```diff
@@ -62,6 +63,15 @@ const functionsMap: FunctionMap[] = [
   ],
 ] as any
 
+export interface UseFullscreenOptions extends ConfigurableDocument {
+  /**
+   * Automatically exit
```

Comment:
Maybe a more descriptive doc like "Automatically exit fullscreen when component is unmounted"